packages/sirius-web/backend/sirius-web-tests/src/main/java/org/eclipse/sirius/web/tests/services/representation/RepresentationMetadataLabelVerifier.java

```diff
@@ -0,0 +1,61 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.web.tests.services.representation;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.fail;
+
+import com.jayway.jsonpath.JsonPath;
+
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.function.Consumer;
+
+import org.eclipse.sirius.components.representations.IRepresentation;
+import org.eclipse.sirius.web.application.UUIDParser;
+import org.eclipse.sirius.web.tests.graphql.RepresentationMetadataQueryRunner;
+import org.springframework.stereotype.Service;
+
+/**
+ * Used to verify the label of a representation thanks to its metadata.
+ *
+ * @author gcoutable
+ */
+@Service
+public class RepresentationMetadataLabelVerifier {
```

Comment:
This concern is only relevant when we are testing the title expression and renaming representations, it is not relevant in other tests

Reply:
Done